module/purpose/PS3Walk/src/PS3Walk.cpp

```diff
@@ -33,32 +34,42 @@
 #include "message/strategy/StandStill.hpp"
 
 #include "utility/input/LimbID.hpp"
+#include "utility/skill/Script.hpp"
 
 namespace module::purpose {
 
+
     using extension::Configuration;
+    using extension::behaviour::Task;
+
+    using message::actuation::LimbsSequence;
     using message::behaviour::state::Stability;
     using message::behaviour::state::WalkState;
     using message::skill::Kick;
     using message::skill::Look;
     using message::skill::Walk;
     using message::strategy::FallRecovery;
     using message::strategy::StandStill;
+
     using utility::input::LimbID;
+    using utility::skill::load_script;
 
+    PS3Walk::PS3Walk(std::unique_ptr<NUClear::Environment> environment) : Reactor(std::move(environment)) {
+
+        on<Configuration>("PS3Walk.yaml").then([this](const Configuration& config) {
+            // Use configuration here from file PlanKick.yaml
+            this->log_level                 = config["log_level"].as<NUClear::LogLevel>();
+            cfg.maximum_forward_velocity    = config["maximum_forward_velocity"].as<double>();
+            cfg.maximum_rotational_velocity = config["maximum_rotational_velocity"].as<double>();
+        });
 
-    PS3Walk::PS3Walk(std::unique_ptr<NUClear::Environment> environment) : BehaviourReactor(std::move(environment)) {
 
+        // Start the Director graph for the KeyboardWalk.
         on<Startup>().then([this] {
             // At the start of the program, we should be standing
-            // Without these emis, modules that need a Stability and WalkState messages may not run
+            // Without these emits, modules that need a Stability and walkstate messages may not run
             emit(std::make_unique<Stability>(Stability::UNKNOWN));
             emit(std::make_unique<WalkState>(WalkState::State::STOPPED));
-
-            // On the lowest level, just stand
-            emit<Task>(std::make_unique<StandStill>(), 0);
-            // On the highest level, recover from falling
-            emit<Task>(std::make_unique<FallRecovery>(), 3);
```

Comment:
How come these are removed?

Reply:
We wanted to robot to freeze after doing the moves. I'll talk to tom and confirm this is correct.

Reply:
Unless the robot has fallen over, fall recovery shouldn't stop the robot from freezing after the dance moves.

Reply:
I still feel like these should be here. Falling shouldn't affect anything and is nice to have as another thing to show off at events (we're often asked if they can get up by themselves). Standing is useful to have in the background but I guess can be left out but might cause some awkward situations. The dancing can be a mode that just doesn't have the stand in the background. As we have walk and head as modes, I think we should also have dancing as a mode.

Reply:
Iirc, the problem was once a dance move finishes, stand just tries to take over priority again

Reply:
Yes but that can be fixed by having a dance mode and not having stand active during the dance mode